M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -677,8 +672,8 @@ doc ///
 	     @HREF("https://www.tandfonline.com/doi/abs/10.1080/10586458.2018.1473821",
 		       "Cohen-Lenstra heuristics for torsion in homology of random complexes")@
 	     by  M. Kahle, F. H. Lutz, A. Newman, and K. Parsons [Exp. Math. vol. 29, no. 3 (2020)].
-	     When the option Verify is set to true, the output of the method will produce  complexes which have exactly 
-	     $m$ faces of dimension $d$.
+             By default, there is a chance that the putoput may have fewer than m faces. To avoid this, set
+             the Verify option to true.
```

Comment:
putoput -> output

You may also want to use dollar signs around the "m" so it's TeX-ified like the surrounding docs.